gobblin-compaction/build.gradle

```diff
@@ -0,0 +1,58 @@
+apply plugin: 'java'
+apply plugin: 'eclipse'
+
+dependencies {
+  if (project.hasProperty('useHadoop2')) {
+    compile externalDependency.hadoopCommon
+    compile externalDependency.hadoopClientCore
+    compile externalDependency.hadoopHdfs
+    compile externalDependency.hadoopAuth
+  } else {
+    compile externalDependency.hadoop
+  }
+  compile externalDependency.commonsLang
+  compile externalDependency.hiveService
+  compile externalDependency.hiveJdbc
+  compile externalDependency.hiveMetastore
+  compile externalDependency.hiveExec
+  compile externalDependency.antlrRuntime
+  compile externalDependency.derby
+  compile externalDependency.jdo2
+  compile externalDependency.datanucleusCore
+  compile externalDependency.datanucleusRdbms
+}
```

Comment:
Looks like a majority of the above jar files are actually just runtime dependencies, and not compile time dependencies. Can you change them